tools/MINGW-packages/mingw-w64-imagemagick/PKGBUILD

```diff
@@ -12,23 +12,15 @@
 pkgver=6.9.11.16
 
 _realname=imagemagick
-pkgbase="mingw-w64-${_realname}"
-pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgbase="mingw-w64-natron_${_realname}"
+pkgname="${MINGW_PACKAGE_PREFIX}-natron_${_realname}"
 pkgrel=5
 pkgdesc="An image viewing/manipulation program (mingw-w64)"
 arch=('any')
 url="https://www.imagemagick.org/"
 license=("custom")
-makedepends=(
-             "${MINGW_PACKAGE_PREFIX}-libpng"
-             "${MINGW_PACKAGE_PREFIX}-bzip2"
-             "${MINGW_PACKAGE_PREFIX}-xz"
-             "${MINGW_PACKAGE_PREFIX}-zlib"
-             "${MINGW_PACKAGE_PREFIX}-freetype"
-             "${MINGW_PACKAGE_PREFIX}-fontconfig"
-	     "${MINGW_PACKAGE_PREFIX}-lcms2"
-	     "${MINGW_PACKAGE_PREFIX}-libheif"
-            )
+makedepends=("${MINGW_PACKAGE_PREFIX}-autotools"
```

Comment:
It looks like a bunch of the libraries in the makedepends section didn't need to be there because they are already in the depends section. Added a few missing build dependencies to enable building from a clean MSYS2 install.